### PR TITLE
fix: Added factories & billing model additions

### LINKED ISSUE #6127: Additional fields required for timezone & invoice sent out date
**Is your feature request related to a problem? Please describe.**
Ref: https://github.com/fossasia/open-event-frontend/issues/3126
These need to be added for the FE form to work.

**Additional context**
working on it.

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #6127 

ref: https://github.com/fossasia/open-event-frontend/pull/3152 

#### Short description of what this resolves:
- Added factories for admin billing details
- Added `invoice_sending_day` & `invoice_sending_timezone`
___________

**A field is required billing_state as well**
ref: https://github.com/fossasia/open-event-frontend/pull/3215#pullrequestreview-256314546
- `admin_billing_state` for admin billing details(to pay event invoice)
- `biling_state` for user billing details(to pay event invoice)

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/schema/settings.py [hunk 1]
```diff
@@ -200,10 +200,15 @@ class Meta:
     smtp_port = fields.Integer(allow_none=True)
     smtp_encryption = fields.Str(allow_none=True)  # Can be tls, ssl, none
 
+    # Event Invoices settings
+    invoices_sent_day = fields.Integer(allow_none=False, default=1)
+    invoices_timezone = fields.Str(allow_none=False, default="UTC")
```
Comment (iamareebjamal) [inline app/api/schema/settings.py]: Why is this needed?

Comment (mrsaicharan1) [reply app/api/schema/settings.py]: TO specify on what day the event invoice should be sent every month.

Comment (iamareebjamal) [reply app/api/schema/settings.py]: timezone

Comment (mrsaicharan1) [reply app/api/schema/settings.py]: Suppose we are sending it on the first of every month, it is used to specify in which timezone  it should be sent.(1st January in IST or PDT)

Comment (mrsaicharan1) [reply app/api/schema/settings.py]: https://github.com/fossasia/open-event-frontend/issues/3126#issue-455093188

**Comment by mrsaicharan1** [comment]
@uds5501 @shreyanshdwivedi Please have a look here too.

File: app/api/schema/settings.py (hunk 1 above)
Comment (iamareebjamal) [reply app/api/schema/settings.py]: invoices_sent_day -> invoice_sending_{day, timezone}

Comment (mrsaicharan1) [reply app/api/schema/settings.py]: @iamareebjamal Done.

File: app/api/schema/settings.py [hunk 2]
```diff
@@ -200,10 +200,15 @@ class Meta:
     smtp_port = fields.Integer(allow_none=True)
     smtp_encryption = fields.Str(allow_none=True)  # Can be tls, ssl, none
 
+    # Event Invoices settings
+    invoices_sending_day = fields.Integer(allow_none=False, default=1)
```
Comment (iamareebjamal) [inline app/api/schema/settings.py]: Make singular `invoice_sending_day`

File: app/api/schema/settings.py [hunk 3]
```diff
@@ -200,10 +200,15 @@ class Meta:
...
+    invoices_sending_timezone = fields.Str(allow_none=False, default="UTC")
```
Comment (iamareebjamal) [inline app/api/schema/settings.py]: Same

File: app/api/schema/settings.py (hunk 2 above)
Comment (mrsaicharan1) [reply app/api/schema/settings.py]: @iamareebjamal Done.

File: app/api/schema/settings.py (hunk 3 above)
Comment (mrsaicharan1) [reply app/api/schema/settings.py]: @iamareebjamal Done.[](url)

**Comment by mrsaicharan1** [comment]
@iamareebjamal @shreyanshdwivedi @uds5501 Please review.

**Comment by mrsaicharan1** [comment]
@iamareebjamal Please merge this as this is a blocker for https://github.com/fossasia/open-event-frontend/pull/3152
